wger/exercises/models.py

```diff
@@ -175,7 +175,7 @@ class Exercise(AbstractSubmissionModel, AbstractLicenseModel, models.Model):
 
     name = models.CharField(max_length=200,
                             verbose_name=_('Name'))
-    '''The exercise's name, with correct upercase'''
+    '''The exercise's name, with correct uppercase'''
```

Comment:
Is there a need for a block comment on a single line comment? Great job caching the typo error here.

Reply:
Sphinx uses docstrings to generate docs